venice/templates/venice-router/router-deployment.yaml

```diff
@@ -34,7 +34,58 @@ spec:
           command: [ "sh", "-c" ]
           args:
             - >
-              env &&
+              env && 
+              set -e &&
+              setup_tls() {
+                name=venice
+                crtFile=/venice/certs/tls.crt
```

Comment:
is it expected that the names "tls.crt", "tls.key" are always the same ?
we make the name of the secret configurable but the file names may be different